bin/wrappers.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+_fk_wrapper_open()
+{
+    case $(uname) in
+        "Darwin")
+            open $@ &> /dev/null
+            ;;
+        "Linux")
+            xdg-open $@ &> /dev/null
+            ;;
+        *)
+            echo "Sorry but we don't support $(uname) yet."
```

Comment:
:joy:

Reply:
Add an instruction for the user to create an issue here :joy: